### PR TITLE
fix playground/graphiql on lambda

### PR DESCRIPTION
**Pull request by lookapanda**
As suggested: https://github.com/serverless/serverless-graphql/issues/256#issuecomment-364470446

Tested and works: https://7z9jwa60yg.execute-api.us-east-1.amazonaws.com/production/playground

### CODE REVIEW

**Comment by sid88in** [comment]
awesome! thanks @lookapanda

File: app-backend/dynamodb/handler.js [hunk 1]
```diff
@@ -23,14 +23,20 @@ exports.graphqlHandler = function graphqlHandler(event, context, callback) {
 };
 
 // for local endpointURL is /graphql and for prod it is /stage/graphql
-exports.playgroundHandler = lambdaPlayground({
-  endpoint: process.env.REACT_APP_GRAPHQL_ENDPOINT
-    ? process.env.REACT_APP_GRAPHQL_ENDPOINT
-    : '/production/graphql',
-});
+exports.playgroundHandler = (event, context, callback) => {
```
Comment (sid88in) [inline app-backend/dynamodb/handler.js]: dont think we need this

File: app-backend/rest-api/handler.js [hunk 2]
```diff
@@ -23,14 +23,20 @@ exports.graphqlHandler = function graphqlHandler(event, context, callback) {
 };
 
 // for local endpointURL is /graphql and for prod it is /stage/graphql
-exports.playgroundHandler = lambdaPlayground({
-  endpoint: process.env.REACT_APP_GRAPHQL_ENDPOINT
-    ? process.env.REACT_APP_GRAPHQL_ENDPOINT
-    : '/production/graphql',
-});
+exports.playgroundHandler = (event, context, callback) => {
```
Comment (sid88in) [inline app-backend/rest-api/handler.js]: dont think we need this

File: app-backend/dynamodb/handler.js (hunk 1 above)
Comment (lookapanda) [reply app-backend/dynamodb/handler.js]: Need what?

**Comment by lookapanda** [comment]
I just assumed it isn't working for the others aswell, I was using RDS when I noticed it isn't working. For testing I had to use my private AWS account and I don't have a running RDS instance, sorry :P

File: app-backend/dynamodb/handler.js (hunk 1 above)
Comment (sid88in) [reply app-backend/dynamodb/handler.js]: I mean I don't think we need these changes in rest and dynamodb backend. GraphiQL and Playground are working with those backends. We only see problem with RDS

**Comment by sid88in** [comment]
Fair enough :) 

1. Will be great if can you please revert back changes for dynamo and rest.
2. Did you test this PR (for RDS) on a running RDS instance or local ?

**Comment by lookapanda** [comment]
I didn't test this particular PR for RDS, but as I said, on our project, we're using it with RDS and I had to do this change to make it work. It was already running locally, it's only an issue on the actual Lambda.

**Comment by sid88in** [comment]
correct! ok sounds good :) Lets merge this PR once you revert back changes for dynamo and rest.

**Pull request merged**